runbotics-orchestrator/src/main/java/com/runbotics/repository/UserRepository.java

```diff
@@ -47,5 +47,5 @@ public interface UserRepository extends JpaRepository<User, Long>, JpaSpecificat
 
     Page<User> findAllByActivatedIsFalseAndEmailIsContaining(Pageable pageable, String email);
 
-    Optional<User> findOneByEmailOrLogin(String email, String login);
+    Optional<User> findOneByIdNotAndEmailOrIdNotAndLogin(Long id1, String email, Long id2, String login);
```

Comment:
That is because if we create the same logic expression in a different way then it didn't work so I had to use two times the same id. That named query can be replaced with standard query and perhaps it would be a simpler option. If there is a better solution to this then I'm open to it.

Reply:
create your own query